File: LeetCode/problems/3110-score-of-a-string/3110-score-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    let score = 0;
    for(let L = 0; L<s.length-1; L++){
        score += Math.abs(s.charCodeAt(L)-s.charCodeAt(L+1));
    }
    return score;
};

File: LeetCode/problems/3110-score-of-a-string/3110-score-of-a-string.rs
impl Solution {
    pub fn score_of_string(s: String) -> i32 {
        let mut score: i32 = 0;
        let mut prev: i32 = 0;
        for c in s.chars(){
            score += (c as i32 - prev).abs();
            prev = c as i32;
        }
        score -= s.chars().next().unwrap() as i32;
        score
    }
}
